code/data_functions.py

```diff
@@ -271,9 +254,86 @@ def clean_IRENA_regression(param, paths):
                                                IRENA_summary.loc[(country, 'Concentrated solar power'), 'FLH (h)']]
         except KeyError:
```

Comment:
what happens if I run the code with custom subregions?
will it stop each time it does not find the key of a subregion?
what if we return the list of subregions that are missing?